CONTRIBUTING.md

```
@@ -178,10 +163,64 @@ Enhancement suggestions are tracked as [GitHub issues](https://github.com/uncles
 ### Your First Code Contribution
 
-<!-- TODO
-include Setup of env, IDE and typical getting started instructions?
+#### Development Environment Setup
 
--->
+1. **Clone the repository**:
+
+   ```bash
+   git clone https://github.com/unclesp1d3r/gold_digger.git
+   cd gold_digger
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)